testing/vcs/test_boxfill_decreasing_latitude.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
+import cdms2
+import cdutil
+import os
+import sys
+import vcs
+
+pth = os.path.join(os.path.dirname(__file__),"..")
```

Comment:
@danlipsa It would be good if new code followed PEP8 convention.

In this case, there should be a space after the comma.